_sass/leaderboard.scss

```diff
@@ -0,0 +1,50 @@
+.leaderboard {
+  // padding: 40px 30px;
+  max-width: 800px;
+  margin: auto;
+  margin-bottom: 80px;
+  // background-color: #133667 !important;
```

Comment:
If you don't need this styling I'd remove it. No need to push comments code. If you are saving it for a future PR, I'd move it to another file and checkout the `git stash` command.

Reply:
I've removed commented code